app/konnect/service-catalog/integrations/gitlab.md

```diff
@@ -19,11 +19,31 @@ For each linked project, the UI can show a **Project Summary** with simple data
 * Only [GitLab.com subscriptions](https://docs.gitlab.com/ee/subscriptions/gitlab_com/) are supported at this time
 
 ## Authorize the GitLab integration
-
+{% navtabs %}
+{% navtab Saas %}
 1. From the **{% konnect_icon servicehub %} Service Catalog** in {{site.konnect_product_name}}, select **[Integrations](https://cloud.konghq.com/us/service-catalog/integrations)**. 
-2. Select **GitLab**, then **Install GitLab**.
-3. Click **Authorize**. 
-
+1. Select **GitLab**, then **Install GitLab**.
+1. Click **Authorize**. 
+
+{% endnavtab %}
+{% navtab Self-hosted %}
+To use the GitLab integration in a self-hosted environment:
+
+1. [Create a group-owned application](https://docs.gitlab.com/integration/oauth_provider/) in your GitLab instance. This is required to enable OAuth access for your organization.
+   * Use your {{site.konnect_short_name}} instance’s  URL as the redirect URI in Gitlab.
+   * Make sure the application has the `api` scope.
+1. In the {{site.konnect_short_name}} UI, navigate to the [GitLab integration](https://cloud.konghq.com/us/service-catalog/integrations/gitlab/configuration)
```

Comment:
```suggestion
1. In the {{site.konnect_short_name}} UI, navigate to the [GitLab integration](https://cloud.konghq.com/service-catalog/integrations/gitlab/configuration)
```

Reply:
@Guaris @cloudjumpercat I _think_ the geo does need to be in the URL
![Screenshot 2025-04-23 at 11 35 04 AM](https://github.com/user-attachments/assets/31be33d6-3492-4472-a16f-0e710e670389)